gusto/configuration.py

```diff
@@ -93,20 +95,19 @@ class EadyParameters(Configuration):
     f = None
 
 
-class CompressibleEadyParameters(Configuration):
+class CompressibleEadyParameters(CompressibleParameters):
 
     """
     Physical parameters for Compressible Eady
     """
+    eady_parameters = EadyParameters()
+    Nsq = eady_parameters.Nsq
```

Comment:
I think that rather than doing this, you can add EadyParameters in the class definition:
`class CompressibleEadyParameters(CompressibleParameters, EadyParameters)`